WordParserIntertec/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿/*
+Write a program that parses a sentence and replaces each word with the following: 
+1) The first letter of the word
+2) The number of distinct characters between first and last character
+3) The last letter of the word. 
+For example, Smooth would become S3h. 
+Words are separated by spaces or non-alphabetic characters and these separators should be maintained in their original form and location in the answer. 
+A few of the things we will be looking at is accuracy, efficiency, solution completeness. 
+*/
+public class Program
+{
+    public static string WordParser(string input)

```

Comment:
This method is too big, I think we could refactor it

Reply:
Right, can we refactor it together?